src/main/java/org/bookstore/controller/BookController.java

```diff
@@ -29,13 +33,24 @@ public List<BookDto> findAll() {
         return bookService.findAll();
     }
 
+    @ResponseStatus(HttpStatus.NO_CONTENT)
     @GetMapping("/{id}")
     public BookDto getBookById(@PathVariable Long id) {
         return bookService.getBookById(id);
     }
 
-    @GetMapping("/by-title")//by-name?title=bob
-    public List<BookDto> getAllByName(@RequestParam String title) {
-        return bookService.getAllByName(title);
+    @GetMapping("/title")
```

Comment:
Not the best endpoint. localhost:8080/title?title="Sherlock". there are two `title` in uri